requirements/edx/github.in

```diff
@@ -97,6 +97,8 @@ git+https://github.com/edx/crowdsourcehinter.git@2178ac72891392106ffef389651aef3
 # Temporary package pointed to Pearson-Advanced, we don't need to consider this change for version upper Maple.
 -e git+https://github.com/Pearson-Advance/xblock-lti-consumer.git@75a21d42626c6a72dc3435ed0602a1403a918b75#egg=lti_consumer-xblock
 
+# Pearson Custom version of edx enterprise
+git+https://github.com/Pearson-Advance/edx-enterprise.git@pearson-release/juniper.master#egg=edx-enterprise==3.2.14.post1
```

Comment:
Does it need the '-e' flag? @diegomillan

Reply:
@Squirrel18 it's not required because I'm making sure `pip` will read the right version.